### ISSUE TITLE
test_aio_suspend crashes on Linux with rust nightly

### ISSUE THREAD

**Issue by asomers** [nix-rust/nix#1099]
With the latest nightly compiler, Travis's Linux builds (but not my local linux builds) fail with this error:
```
../sysdeps/pthread/aio_suspend.c:102: do_aio_misc_wait: Assertion `status == 0 || status == -11' failed
```
That error is coming from within glibc itself.

**Comment by asomers** [comment]
On Cirrus-CI this test is now failing with a different error: `The futex facility returned an unexpected error code.`.  But it's still a glibc bug.

**Comment by SteveLauC** [comment]
Seems that this issue has been solved, close as completed in #2262, but if it still occurs in the future, we can reopen it at any time.
